LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -27,6 +27,48 @@ public enum GlucoseStoreResult<T> {
     case failure(Error)
 }
 
+public protocol GlucoseStoreProtocol {
```

Comment:
This protocol was added so it's possible to create mock stores to use with LoopDataManager